codecov.yml

```diff
@@ -1,2 +1,2 @@
 ignore:
-  - "lib/.*"
+  - 'lib/.*'
```

Comment:
Do we have a `lib` folder?

Reply:
Good catch! We actually don't need this file.

Reply:
It will be removed in your codecov branch